src/api/migrations/0002_auto_20180701_0923.py

```diff
@@ -0,0 +1,84 @@
+import datetime
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+import django.utils.timezone
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('api', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='CheckLicenseFileUpload',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('created', models.DateTimeField(auto_now_add=True)),
+                ('file', models.FileField(upload_to=api.models.user_directory_path)),
+                ('result', models.CharField(max_length=128)),
+                ('status', models.IntegerField(default=200)),
+                ('owner', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL)),
+            ],
+        ),
+        migrations.AddField(
+            model_name='comparefileupload',
+            name='message',
+            field=models.CharField(default=datetime.datetime(2018, 7, 1, 9, 23, 13, 43252), max_length=64),
+            preserve_default=False,
+        ),
+        migrations.AddField(
+            model_name='comparefileupload',
+            name='status',
+            field=models.IntegerField(default=200),
+        ),
+        migrations.AddField(
+            model_name='convertfileupload',
+            name='message',
+            field=models.CharField(default=django.utils.timezone.now, max_length=64),
+            preserve_default=False,
+        ),
+        migrations.AddField(
+            model_name='convertfileupload',
+            name='status',
+            field=models.IntegerField(default=200),
+        ),
+        migrations.AddField(
+            model_name='convertfileupload',
+            name='tagToRdfFormat',
+            field=models.CharField(blank=True, max_length=16, null=True),
+        ),
+        migrations.AddField(
+            model_name='validatefileupload',
+            name='status',
+            field=models.IntegerField(default=200),
+        ),
+        migrations.AlterField(
+            model_name='comparefileupload',
+            name='file1',
+            field=models.FileField(upload_to=api.models.user_directory_path),
+        ),
+        migrations.AlterField(
+            model_name='comparefileupload',
+            name='file2',
+            field=models.FileField(upload_to=api.models.user_directory_path),
+        ),
+        migrations.AlterField(
+            model_name='convertfileupload',
+            name='file',
+            field=models.FileField(upload_to=api.models.user_directory_path),
+        ),
+        migrations.AlterField(
+            model_name='validatefileupload',
+            name='file',
+            field=models.FileField(upload_to=api.models.user_directory_path),
+        ),
+    ]
```

Comment:
Can you please not commit all the local migrations files. Just start the start the migrations from the beginning and just commit the 0001_initial.py.